src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(Map<String, Integer> storage, FruitTransaction transaction) {
+        storage.put(transaction.getFruit(), transaction.getQuantity());
```

Comment:
```suggestion
    public void handle(FruitTransaction transaction) {
        Storage.storage.put(transaction.getFruit(), transaction.getQuantity());
```
Fix it for other handlers